frontends/web/src/routes/account/send/send.tsx

```diff
@@ -161,34 +163,31 @@ class Send extends Component<Props, State> {
         this.setState({ coinControl: !!(config.frontend || {}).coinControl });
       }
     });
-    this.unsubscribe = apiWebsocket((payload) => {
-      if ('type' in payload) {
-        const { data, meta, type } = payload;
-        switch (type) {
-        case 'device':
-          switch (data) {
-          case 'signProgress':
-            this.setState({ signProgress: meta, signConfirm: false });
-            break;
-          case 'signConfirm':
-            this.setState({ signConfirm: true });
-            break;
-          }
-          break;
-        case 'account':
-          switch (data) {
-          case 'syncdone':
-            if (this.props.code) {
-              accountApi.getBalance(this.props.code)
-                .then(balance => this.setState({ balance }))
-                .catch(console.error);
+
+    this.unsubscribeList = [
+      apiWebsocket((payload) => {
+        if ('type' in payload) {
+          const { data, meta, type } = payload;
+          switch (type) {
+          case 'device':
+            switch (data) {
+            case 'signProgress':
+              this.setState({ signProgress: meta, signConfirm: false });
+              break;
+            case 'signConfirm':
+              this.setState({ signConfirm: true });
+              break;
```

Comment:
I'll add typed subscriptions for devicesync.signProgress and devicesync.signConfrim in a later commit or PR.